pkg/controller/publishingstrategy/publishingstrategy_controller.go

```diff
@@ -363,6 +393,49 @@ func validateStaticSpec(ingressController ingresscontroller.IngressController, d
 	return true
 }
 
+func (r *ReconcilePublishingStrategy) ensureAWSLoadBalancerType(reqLogger logr.Logger, ic *ingresscontroller.IngressController, ps v1alpha1.PublishingStrategy) (result reconcile.Result, err error) {
```

Comment:
Unfortunately, I don't think there's a clean way to do this (without at least a significant refactor) because we don't support the equivalent in GCP. Additionally, this is a feature that needs to get out ASAP, while also balancing the fact that we want to shut down cloud-ingress-operator in the near future. If you have a code suggestion though, I'd be happy to review it

Reply:
Unfortunately, I don't think there's a clean way to do this (without at least a significant refactor) because we don't support the equivalent in GCP. There's no actual AWS API calls happening here though, just a check based on if the underlying platform in AWS or not, but the logic only depends on the kubernetes resources. Additionally, this is a feature that needs to get out ASAP, while also balancing the fact that we want to shut down cloud-ingress-operator in the near future. If you have a code suggestion though, I'd be happy to review it

Reply:
Sounds good. Let's do the refactor when we have more time.

Reply:
created https://issues.redhat.com/browse/OSD-11933